analyzers/src/SonarAnalyzer.CSharp/Rules/CheckArgumentException.cs

```diff
@@ -101,7 +101,13 @@ private static void CheckForIssue(SyntaxNodeAnalysisContext analysisContext)
             }
 
             var methodArgumentNames = GetMethodArgumentNames(objectCreationSyntax);
-            if (methodArgumentNames.Contains(TakeOnlyBeforeDot(parameterNameValue)))
+            var parameterName = TakeOnlyBeforeDot(parameterNameValue);
+            if (string.IsNullOrWhiteSpace(parameterName))
```

Comment:
We should use IsNullOrEmpty to create less FNs
```suggestion
            if (string.IsNullOrEmpty(parameterName))
```